pkg/controller/cloneset/scale/cloneset_scale.go

```diff
@@ -31,15 +29,15 @@ const (
 
 // Interface for managing replicas including create and delete pod/pvc.
 type Interface interface {
-	ManageReplicas(
+	Manage(
```

Comment:
Can we change Interface to other name? say ControlInterface?

Reply:
This is a common usage in Golang, when a package only provides this interface to other users.

You may find hundreds of it, if you grep `type Interface interface` in Kubernetes repository :)